.eslintrc.json

```diff
@@ -29,10 +28,12 @@
     "require-jsdoc": 0,
     "valid-jsdoc": 0,
     "prefer-arrow-callback": 1,
-    "no-var": 1
+    "no-var": 1,
+    "no-undef": "warn"
```

Comment:
Wondering whether it'd be better to error here?

Reply:
`"error"` is possible if we add
```
    "browser": true,
    "jest": true,
```
to `"env"`. Are you ok with these additions? Alternatively, `/* global var1, var2 */` could be added to relevant files.

Error log below for reference:
```
> root@ lint /home/mdmower/source/amp-toolbox
> eslint . && lerna run lint


/home/mdmower/source/amp-toolbox/packages/cache-url/lib/browser/Sha256.js
  25:10  error  'crypto' is not defined  no-undef

/home/mdmower/source/amp-toolbox/packages/cache-url/spec/AmpUrlSpec.js
  23:20  error  'window' is not defined  no-undef
  24:26  error  'window' is not defined  no-undef

/home/mdmower/source/amp-toolbox/packages/cli/spec/cliSpec.js
  25:1  error  'test' is not defined  no-undef

/home/mdmower/source/amp-toolbox/packages/cli/spec/ioSpec.js
  21:1  error  'test' is not defined  no-undef
  26:1  error  'test' is not defined  no-undef
  31:1  error  'test' is not defined  no-undef

/home/mdmower/source/amp-toolbox/packages/cors/spec/lib/corsSpec.js
   49:5   error  'caches' is not defined  no-undef
   52:29  error  'caches' is not defined  no-undef
  119:33  error  'caches' is not defined  no-undef
  140:33  error  'caches' is not defined  no-undef
  154:31  error  'caches' is not defined  no-undef
  155:14  error  'jest' is not defined    no-undef

/home/mdmower/source/amp-toolbox/packages/optimizer/spec/lib/HtmlDomHelper.test.js
  21:1  error  'test' is not defined  no-undef
  31:1  error  'test' is not defined  no-undef

✖ 15 problems (15 errors, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@ lint: `eslint . && lerna run lint`
npm ERR! Exit status 1
```